aws_adfs/login.py

```diff
@@ -151,7 +155,7 @@ def _authenticate(config, username=None, password=None):
     del password
 
     # Decode the response
-    html = ET.fromstring(response.text.decode('utf8'), ET.HTMLParser())
+    html = ET.fromstring(response.text, ET.HTMLParser())
```

Comment:
I'll create a docker container to verify that one. Wonder if it'll be sufficient to confirm that the bug exists and them was fixed.